packages/console/src/components/ActionMenu/ActionMenuButton.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { forwardRef } from 'react';
+
+import Button, { ButtonProps } from '../Button';
+
+const ActionMenuButton = forwardRef<HTMLDivElement, ButtonProps>((props, anchorReference) => (
+  <div ref={anchorReference} style={{ display: 'inline-block' }}>
```

Comment:
Why do we need inline-block here? Also, why using inline styles instead of class?

Reply:
Is a special "div" only for adding ref without changing original component (hear is Button), so I use inline-style instead of css file.

use "block" to easily calc bounding rect size, and "inline" to keep original button's behavior as much as posible.

Reply:
not sure but, run React cloneElement on the Button, and insert ref prop?

Reply:
yep, I tried this but failed, maybe refactor later?